edit-post/components/header/style.scss

```diff
@@ -42,6 +42,12 @@
 .edit-post-header__settings {
 	display: inline-flex;
 	align-items: center;
+
+	// Ensure that even SVG icons that don't include the .dashicon class are colored
+	.components-icon-button svg {
```

Comment:
I think this is no longer necessary?

Reply:
My golly, of course you're right. Removed!